website/docs/integrations/home_assistant.md

```diff
@@ -395,7 +395,7 @@ tesla_location:
       unit_of_measurement: ft
       icon_template: mdi:image-filter-hdr
       value_template: >
-       {{ (states('sensor.tesla_elevation') | float * 3.2808 ) | round(2) }}
+       {{ (states('sensor.tesla_elevation') | float * 3.2808) | round(2) }}
```

Comment:
This space doesn't matter in Jinja2. You can try this in your 'Developer Tools > Template'. Just paste these two lines there and you'll see it's the same outcome:
```
{{ ('1.5' | float * 3.2808) | round(2) }}
{{ ('1.5' | float * 3.2808 ) | round(2) }}
```

Reply:
I think you are right, I cannot reproduce it any more now, probably was due to something else.